Dependencies/napi/CMakeLists.txt

```diff
@@ -90,10 +50,51 @@ endif()
 
 target_include_directories(napi PUBLIC "include")
 
-if(NAPI_JAVASCRIPT_ENGINE STREQUAL "Chakra")
-    target_link_libraries(napi PUBLIC "chakrart.lib")
-elseif(NAPI_JAVASCRIPT_ENGINE STREQUAL "V8")
-    target_link_libraries(napi PUBLIC v8)
-endif()
+# TODO: Make this block less weird.
```

Comment:
what do you mean by this?

Reply:
```suggestion
```

Reply:
Outdated. This block used to be weirder, but it's now less weird, and make it not weird at all is no longer an ambition of this change.